src/app/services/xml-parsers/app-parser.ts

```diff
@@ -52,8 +52,7 @@ export class RdgParser extends EmptyParser implements Parser<XMLElement> {
         return !Array.from(attributes).some(({ name, value }) => notSignificantReading.includes(`${name}=${value}`));
     }
 }
-
-@xmlParser('app', AppParser)
+@xmlParser('evt-apparatus-entry-parser', AppParser)
```

Comment:
Why this change in tagName definition?

Reply:
@ChiaraDipi I think I changed it to use the new `ParserRegister.get` function. See commit 6a6494d